app/src/features/projects/view/components/FundingSource.tsx

```diff
@@ -19,24 +30,111 @@ const useStyles = makeStyles({
   }
 });
 
-export interface IProjectDetailsProps {
+export interface IProjectFundingProps {
   projectForViewData: IGetProjectForViewResponse;
+  codes: IGetAllCodeSetsResponse;
+  refresh: () => void;
 }
 
 /**
  * Funding source content for a project.
  *
  * @return {*}
  */
-const FundingSource: React.FC<IProjectDetailsProps> = (props) => {
+const FundingSource: React.FC<IProjectFundingProps> = (props) => {
   const {
-    projectForViewData: { funding }
+    projectForViewData: { funding, id },
+    codes: codes
```

Comment:
nit: change this line to
```suggestion
    codes
```